js/tweek-local-cache/src/tweek-repository.ts

```diff
@@ -205,12 +206,29 @@ export default class TweekRepository {
     return this.observe('_');
   }
 
+  private _dirtyRefresh() {
+    this._isDirty = true;
+    this._rollRefresh();
+  }
+
   private _rollRefresh() {
-    this._refreshPromise = this._refreshKeys();
+    if (this._refreshInProgress || !this._isDirty) return this._refreshPromise;
+
+    this._refreshInProgress = true;
+
+    this._refreshPromise = delay(this._refreshInterval).then(() => this._refreshKeys());
```

Comment:
maybe change the name of `_refreshInterval` to `_refreshDelay` because it's not an interval anymore

Reply:
since this is passed into the constructor, do you think I should rename it there as well? or maybe accept it in either form?  Don't want to accidentally break people if they update

Reply:
I think we can accept both
@Yshayy what do you think?